_data/changelogs/typography.yml

```diff
@@ -0,0 +1,35 @@
+title: Typography
+type: component
+# Provide url for the package CHANGELOG.md. Note: Not all pages will have a related CHANGELOG.md file.
+changelogURL:
+# Add a list of changelog items with newest items first. Uncomment the 'items' data field when adding the first item.
+# Start changelog item. Copy this section to create a new changelog item.
+# items:
+  # - date:
+  #   summary:
+  #   summaryAdditional:
+  #   isBreaking:
+  #   affectsAccessibility:
+  #   affectsAssets:
+  #   affectsGuidance:
+  #   affectsJavascript:
+  #   affectsMarkup:
+  #   affectsStyles:
+  #   githubPr:
+  #   githubRepo:
+  #   versionUswds:
+
```

Comment:
I went ahead and removed these comments because I actually found them more confusing than helpful. Open to differing viewpoints though :)

Reply:
Yeah, it seems only useful in the beginning. After adding updates it just adds noise. 

I left them in originally because I thought some keys would be left out. Instead, I've just left in all keys by default and set to false.